src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -346,99 +341,141 @@ public void fillUpExistingHoneyFrameFromHive(Date currentDate) {
+        }
 
 
-    public void addNewHoneyFrameInHive(Date currentDate) {
+        public void addNewHoneyFrameInHive () {
 
         /*
         this method will add new empty honey frame in hive. If total number of honey frame full in hive is equal with
         6, will call method to collect honey from frames.
          */
 
-        double maxKgOfHoneyPerFrame = 4.5;
-        // a frame could be loaded with around  4.5Kg of honey
+            double maxKgOfHoneyPerFrame = 4.5;
+            // a frame could be loaded with around  4.5Kg of honey
 
-        int honeyFrameFull = 0;
-        for (HoneyFrame honeyFrame : honeyFrames) {
-            if (honeyFrame.getKgOfHoney() == maxKgOfHoneyPerFrame && honeyFrameFull < honeyFrames.size()) {
-                honeyFrameFull += 1;
+            int honeyFrameFull = 0;
+            for (HoneyFrame honeyFrame : honeyFrames) {
+                if (honeyFrame.getKgOfHoney() == maxKgOfHoneyPerFrame && honeyFrameFull < honeyFrames.size()) {
+                    honeyFrameFull += 1;
+                }
             }
-        }
-        if (honeyFrameFull == honeyFrames.size()) {
-            int maximumNumberOfFramesToAdd = 6 - honeyFrameFull;
-            if(maximumNumberOfFramesToAdd!=0){
-                this.setNumberOfHoneyFrame(this.getNumberOfHoneyFrame() + 1);
-                this.addHoneyFrames(createNewHoneyFrame(this.getNumberOfHoneyFrame()));
+            if (honeyFrameFull == honeyFrames.size()) {
+                int maximumNumberOfFramesToAdd = 6 - honeyFrameFull;
+                if (maximumNumberOfFramesToAdd != 0) {
+                    this.setNumberOfHoneyFrame(this.getNumberOfHoneyFrame() + 1);
+                    this.addHoneyFrames(createNewHoneyFrame(this.getNumberOfHoneyFrame()));
+                }
             }
         }
 
-//        System.out.println("Hive ID: " + this.getId());
-//        System.out.println("Honey Frame: " + this.getHoneyFrames());
-//        System.out.println(" date is " + currentDate);
-//        System.out.println(" your hive is :" + this);
-    }
-
 
-    public List<EggsFrame> createNewEggsFrame(int numberOfEggsFrame) {
-        // this method will create a new empty eggs frame
+        public List<EggsFrame> createNewEggsFrame ( int numberOfEggsFrame){
+            // this method will create a new empty eggs frame
 
-        List<EggsFrame> eggsFrames = new ArrayList<>();
-        EggsFrame eggsFrame = new EggsFrame(numberOfEggsFrame, 0);
-        eggsFrames.add(eggsFrame);
-        return eggsFrames;
-    }
+            List<EggsFrame> eggsFrames = new ArrayList<>();
+            EggsFrame eggsFrame = new EggsFrame(numberOfEggsFrame, 0);
+            eggsFrames.add(eggsFrame);
+            return eggsFrames;
+        }
 
-    public List<HoneyFrame> createNewHoneyFrame(int numberOfHoneyFrame) {
-        // this method will create a new empty honey frame
+        public List<HoneyFrame> createNewHoneyFrame ( int numberOfHoneyFrame){
```

Comment:
Looks like `numberOfHoneyFrame` is not used inside this method. Either use it or remove it.